sourced/ml/__main__.py

```diff
@@ -139,8 +138,8 @@ def add_default_args(my_parser):
         help="Combine row and column embeddings together and write them to an .asdf.",
         formatter_class=ArgumentDefaultsHelpFormatterNoNone)
     id2vec_postproc_parser.set_defaults(handler=postprocess_id2vec)
-    id2vec_postproc_parser.add_argument("swivel_output_directory")
-    id2vec_postproc_parser.add_argument("result")
+    id2vec_postproc_parser.add_argument("--swivel-output-directory")
```

Comment:
It was without `--` so that the arguments were free. Now we have to type `--swivel-output-directory` every time. Also they are not required now. Add `-i`, `-o` shorthands, mark as required and add help.

Reply:
ok, I change it just to have the same parameter style in all sourced ml